code/api/src/modules/product_shipments/mutations.js

```diff
@@ -0,0 +1,3 @@
+// Imports
+import { GraphQLInt } from 'graphql'
```

Comment:
I just set up these files, I will be adding a lot to them soon, but need to setup the data so I can test them

Reply:
Sounds good! Thanks for clarifying that.